src/main/kotlin/no/nav/familie/ef/sak/vedtak/domain/Vedtak.kt

```diff
@@ -32,6 +34,8 @@ data class Vedtaksperiode(
 
 data class PeriodeWrapper(val perioder: List<Vedtaksperiode>)
 data class InntektWrapper(val inntekter: List<Inntektsperiode>)
+data class BrevmottakereWrapper(val mottakere: List<Brevmottaker>)
```

Comment:
Her bruker jeg `Brevmottaker` definert i kontrakter. Burde vi ha en egen type i sak for å forenkle fremtidig bakoverkompatibilitet?